log/logClient.go

```diff
@@ -204,6 +205,39 @@ func (fd *Feeder) WatchAlerts(logPath string, jsonFormat bool) error {
 			break
 		}
 
+		if namespace != "" {
```

Comment:
Few comments:
* Compile has to be done just once... not on every event.
* avoid code duplication
* add some unit tests